tools/github_helper/main.go

```diff
@@ -284,12 +281,20 @@ func (h helper) updatePullRequest(pr *github.PullRequest, s *github.CombinedStat
 		state = GH.success
 	}
 
-	if state == GH.failure && (h.Codecov || h.Cla) {
+	if state == GH.failure && (len(h.CheckToSkip) > 0) {
 		privilege := true
 		for _, status := range s.Statuses {
 			if *status.State != GH.success {
-				if !(h.Codecov && strings.HasPrefix(*status.Context, "codecov")) && !(h.Cla && strings.HasPrefix(*status.Context, "cla")) {
-					//Failed, and not eligible for any privilege
+				skip := false
+				for _, check := range h.CheckToSkip {
+					if *status.Context == check {
```

Comment:
I tested its logic by creating a fake combinedStatus directly in the updatePullRequest(), working but naive.
And does this pattern work for you:
keyWord := "cla"   //input flag
pattern := fmt.Sprintf("(^|/)%s(/|$)", keyWord)